pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -95,10 +95,8 @@ func standardDeviation(xs []int) float64 {
 type numaOrSocketsFirstFuncs interface {
 	takeFullFirstLevel()
 	takeFullSecondLevel()
-	takeThirdLevel()
```

Comment:
This is added in the first commit and removed in the second. Can we make the commits additive or squash them?

Reply:
That would be great indeed, maybe the only concern is properly preserving authorship since there are 3 contributors to this PR? Maybe I'm overthinking (can't recall the fine details of the CLA). If this is not a concern and/or can be solved by you folks contributing the change, by all means please squash/rearrange the commits to make them additive!

Reply:
I’ll check with our group but I believe you are correct that this will remove an author, which may not be desired unfortunately